.travis.yml

```diff
@@ -1,45 +1,44 @@
 language: cpp
-compiler: gcc
-git:
-  depth: false
-notifications:
-  email: false
-branches:
-  only:
-    - coverity_scan
-
-env:
-  - CMAKE_GEN="Unix Makefiles"
-    CMAKE_TCH_64="../src/cmake/toolchains/Linux-GNU-64.cmake"
-
-before_install:
-  - echo -n | openssl s_client -connect scan.coverity.com:443 | sed -ne '/-BEGIN CERTIFICATE-/,/-END CERTIFICATE-/p' | sudo tee -a /etc/ssl/certs/ca-
-
-install:
-  - sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-7 60 --slave /usr/bin/g++ g++ /usr/bin/g++-7 
-  - sudo update-alternatives --config gcc
-
-addons:
-  apt:
-    sources:
-      - ubuntu-toolchain-r-test
-    packages:
-      - gcc-7
-      - g++-7
-      - libmysql++-dev
-      
-  coverity_scan:
-    project:
-      name: "Sphereserver/Source-X"
-      description: "Game server for Ultima Online"
-    notification_email: null@null
-    branch_pattern: coverity_scan
-    build_command_prepend: cmake -G '$CMAKE_GEN' -DCMAKE_BUILD_TYPE="Nightly" -DCMAKE_TOOLCHAIN_FILE='$CMAKE_TCH_64' ./src
-    build_command: make -j4
-      
-script:
-  - if [ $COVERITY_SCAN_BRANCH != 1 ]; then travis_terminate 0; fi
-
-after_script:
-  - echo "Printing Coverity log..."
-  - cat cov-int/scm_log.txt
+matrix:
```

Comment:
Are those changes to .travis.yml necessary, considering that we use Travis-CI only for Coverity, and that it's already working just fine?

Reply:
oups i forget to re add Coverity stuff. i have add build for windows and OS X for travis i will re add this for Coverity.

Reply:
I mean, in my opinion we can drop those changes to travis.yml since we are using appveyor for automatic deployment